src/renderer/reader/components/picker/SearchFormPicker.tsx

```diff
@@ -118,6 +120,19 @@ class SearchFormPicker extends React.Component<IProps, IState> {
                             <SVG ariaHidden={true} svg={searchIcon} />
                     }
                 </button>
+                {this.state.inputValue ?
+                <button
+                    className={stylesGlobal.button_clear}
+                    type="reset"
+                    onClick={() => {
+                        this.inputRef.current.value = "";
+                        console.log(this.inputRef.current.value);
```

Comment:
remove logging. but also: does setting the DOM value directly using a ref trigger `onChange={(e: React.ChangeEvent<HTMLInputElement>) => this.setState({ inputValue: e.target.value })}`? if not, we are breaking the React lifecycle of controlled editable components. need to double check...